### ISSUE TITLE
Deprecate Openshift's definitionsFile?

### ISSUE THREAD

**Issue by jwendell** [arquillian/arquillian-cube#741]
Given that we can pass a resource file via `env.config.url` or `env.config.resource.name`; Or even have a file named `kubernetes.json|yml` or `openshift.json|yml` in resources folder, is there a need for such `definitionsFile` configuration parameter for openshift?

Should we deprecate it for now and drop it in a later version?

**Comment by lordofthejars** [comment]
Yes it seems that there is no need for it. Let's deprecate and remove in the feature.
